operatorcourier/validate.py

```diff
@@ -33,16 +33,11 @@ def _bundle_validation(self, bundle):
 
         bundleData = bundle[self.dataKey]
 
-        validationDict[self.crdKey] = self._type_validation(bundleData, self.crdKey, self._crd_validation, False)  
+        validationDict[self.crdKey] = self._type_validation(bundleData, self.crdKey, self._crd_validation, False)
         validationDict[self.csvKey] = self._type_validation(bundleData, self.csvKey, self._csv_validation, True)
         validationDict[self.pkgsKey] = self._type_validation(bundleData, self.pkgsKey, self._pkgs_validation, True)
 
-        valid = True
-        for key, value in validationDict.items():
-            if value is False:
-                valid = False
-        
-        return valid
+        return False not in validationDict.values()
```

Comment:
How is this change related to the malformed yaml? Please limit this pr to a change related to the comments mentioned in the pr and commit message.